### DISCUSSION TITLE
Pull request P08

### DISCUSSION THREAD

**Discussion by ghost** [TheAlbertDev/MASB#59] [Q&A] [answered]
Hello Albert, 

We know it's so late but we have realized know we didn't do the **Pull Request** and then, we remembered that we only created two branches:
- I2C
- SPI
So we don't have a **master** branch to which we can merge our projects. 

![image](https://user-images.githubusercontent.com/78802420/117850510-a72beb80-b285-11eb-95b6-d9744aec69be.png)

What can we do at this point? If we create a new master branch now, it will depend on I2C or SPI branches. But if we don't and we do not do the Pull Request, we won't be able to perform the evaluation and assigned each other as reviewers of the practice. 

Can you guide us? 

Thank you :)

Raimon and Julia.

**Comment by Albert-Alvarez** [comment, marked as answer]
You shouldn't create a master branch and, therefore, you shouldn't make any pull request.
